web/satellite/src/views/registration/RegisterArea.vue

```diff
@@ -808,13 +802,18 @@ async function createUser(): Promise<void> {
 </script>
 
 <style scoped lang="scss">
+    .text-blue {
```

Comment:
Is this class used anywhere?

Reply:
It was used in the title because part of the title was supposed to be blue, but that conflicts with the config which replaces the title. So i left it there to make another update soon for the title.